client/service/commands.go

```diff
@@ -118,33 +171,6 @@ func (a CommandsAPI) waitForCommandFinished(commandID, contextID, clusterID stri
 	}
 }
 
-// Commented till this will be used
-//func (a CommandsAPI) deleteCommand(commandID, contextID, clusterID string) error {
-//	contextDeleteRequest := struct {
-//		CommandID string `json:"commandId,omitempty" url:"commandId,omitempty"`
-//		ContextID string `json:"contextId,omitempty" url:"contextId,omitempty"`
-//		ClusterID string `json:"clusterId,omitempty" url:"clusterId,omitempty"`
-//	}{
-//		CommandID: commandID,
-//		ContextID: contextID,
-//		ClusterID: clusterID,
-//	}
-//	_, err := a.Client.performQuery(http.MethodPost, "/commands/cancel", "1.2", nil, contextDeleteRequest, nil)
-//	return err
-//}
-
-func (a CommandsAPI) deleteContext(contextID, clusterID string) error {
```

Comment:
is deleteContext not used anywhere? If not maybe it should be used to defer delete when finished using this the context with all the commands.